package.json

```diff
@@ -15,9 +15,12 @@
     "node-gyp": "^9.0.0"
   },
   "devDependencies": {
-    "heya-unit": "^0.3.0"
+    "@types/node": "^18.7.18",
+    "heya-unit": "^0.3.0",
+    "typescript": "^4.8.3"
   },
   "scripts": {
+    "pretest": "tsc",
```

Comment:
I am not an expert in Typescript, but I don't see any file produced running this command.